src/libraries/System.Private.Xml/tests/Readers/CharCheckingReader/System.Xml.RW.CharCheckingReader.Tests.csproj

```diff
@@ -11,5 +11,7 @@
     <ProjectReference Include="$(CommonTestPath)System\Xml\ModuleCore\ModuleCore.csproj" />
     <ProjectReference Include="$(CommonTestPath)System\Xml\XmlCoreTest\XmlCoreTest.csproj" />
     <ProjectReference Include="..\..\XmlReaderLib\System.Xml.RW.XmlReaderLib.csproj" />
+
+    <TrimmerRootDescriptor Include="$(ILDescriptorsPath)ILLink.Descriptors.ModuleCore.xml" />
```

Comment:
It looks like `ModuleCore` is really only used by the Xml tests, and not really "common" across a bunch of tests. Can we just put this in the Xml directory, and use a Directory.Build.props/targets file to include it, so all these files don't need to change?

Reply:
1. moved it to xml tests dir
2. It's used by all the `Reader` tests, so we can use it via `Directory.Build.props` for them.
3. Other directories have more projects though, and it didn't seem worth trying to special case them in a top `Directory.Build.*`

Reply:
1. moved it to xml tests
2. All the `Reader` tests use this, so changed to use `Directory.Build.props` for that
3. Only two other tests are using it, so I let them be in the individual project files